lib/elixir/lib/module.ex

```diff
@@ -1,4 +1,9 @@
 defmodule Module do
+  @extra_error_msg_defines? "Use Kernel.function_exported?/3 and Kernel.macro_exported?/3 " <>
+                              "to check for public functions and macros instead"
+
+  @extra_error_msg_definitions_in "Use the Module.__info__/1 callback to get public functions and macros instead"
```

Comment:
Can it be after the `@moduledoc`? Sorry for the nitpicking, as we really have no rules for this.